themes/mongodb/src/.eslintrc.json

```diff
@@ -1,10 +1,12 @@
 {
```

Comment:
Eventually we could look into basing this off of a generally respected style guide like the [airbnb config](https://www.npmjs.com/package/eslint-config-airbnb).

Reply:
I wouldn't be opposed